### DISCUSSION TITLE
Increase range on Zapthrottle Mote Extractor

### DISCUSSION THREAD

**Discussion by Alreyz** [chromiecraft/chromiecraft#7180] [Suggestions]
Plenty of clouds are inaccesible or very difficult to gather. Blizzard made a poor design, but i trust you can do better.

**Comment by cachocarne** [comment]
I don't think the problem is the Mote Extractor range; the issue is that some clouds (particularly in Nagrand) spawn out of reach, way too high in the air or too far away from floating rocks
